### ISSUE TITLE
Version mismatch against v1.1.0

### ISSUE THREAD

**Issue by witcradg** [vdavid/node-mautic#16]
I'm confused. https://www.npmjs.com/package/node-mautic states the project version is 1.1.0.
Installing using npm install node-mautic does, in fact, install a version 1.1.0.
An examination of the node_modules/node-mautic package.json file does indeed show version 1.1.0 as the last line in the package.json file. It has a single dependency: node-fetch.

All good. HOWEVER. The github repo that nodejs.com points to (this repo) contains a tip package.json with a version number of 1.0.3. (on line 3). The dependency listed is for request instead of node-fetch.

What am I missing. I'm looking for the source code for 1.1.0 and I'm not finding it.

**Comment by vdavid** [comment]
That's correct, and I apologize.
I maintain this package but it's just a tiny side project for me, I don't deal with it often.
Last time when I did a change, I did it on a branch, I published it but didn't merge the changes to `master` so you didn't find them.
Thanks for letting me know @witcradg. I merged it, and also took the opportunity to update the dependencies to their latest versions, and published (+pushed!) it all as v1.1.1.

As a _meta_ note: it's really confusing how NPM lets you publish a package without actually pushing the same version to the package `master`. Surely, it's because NPM is completely independent from GitHub, but as they're linking to GitHub, one would expect that they do some sort of verification that it's the same version.
It may even lead to security issues, e.g. one publishes malicious code to NPM and "nice" code to GitHub, perhaps even by the same version, and 99% of users will never notice even if they look for it.

Anyway, this is solved.
